modules/UI/UI.js

```diff
@@ -143,13 +143,22 @@ function setupToolbars() {
  * @see https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API
  */
 UI.toggleFullScreen = function() {
-                            // alternative standard method
-    let isNotFullScreen = !document.fullscreenElement &&
-            !document.mozFullScreenElement && // current working methods
-        !document.webkitFullscreenElement &&
-        !document.msFullscreenElement;
+    let isFullScreen = document.fullscreenElement
+            || document.mozFullScreenElement // current working methods
+            || document.webkitFullscreenElement
+            || document.msFullscreenElement;
 
-    if (isNotFullScreen) {
+    if (isFullScreen) {
+        if (document.exitFullscreen) {
```

Comment:
While doing that we can also do 'requesrFullScreen' for document.documentElement.requestFullscreen. Will make reading this ifs easier